### ISSUE TITLE
Godot can't use dll file

### ISSUE THREAD

**Issue by nongvantinh** [godotengine/godot#39879]
**Godot version:**
 3.2.1

**OS/device including version:**
 Windows 10

**Issue description:**
When import Mysql.Data downloaded from nuget and add it to project Godot doesn't load it.

**Steps to reproduce:**

**Minimal reproduction project:**

**Comment by Calinou** [comment]
@nongvantinh Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by JupiterRider** [comment]
I used your reproduction project and it worked just fine?

How did you add the package? By hand? If so you should run a `nuget restore` or `dotnet restore`

**Comment by nongvantinh** [comment]
No, I installed it using JetBrains Rider and this error happened. I tried to verify that its my fault by create a Console Application and Install Mysql.Data packet from nuget again then past the code in and it works. I don't understand why it's happens with Godot.
![Screenshot (1)](https://user-images.githubusercontent.com/53887662/85934181-69420500-b909-11ea-97ad-374811af44ee.png)

**Comment by JupiterRider** [comment]
Can you even use your Console App to connect to the MySQL database? Cuz that error from your screenshot looks like an Connection error. 

The reason you whole game crashes is, that you should handle exceptions. You gotta catch them 
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/exceptions/

**Comment by nongvantinh** [comment]
No, The console app created in C# 4.7.2 work just fine.
![Screenshot (2)](https://user-images.githubusercontent.com/53887662/85941083-d4f89200-b94a-11ea-8119-28bdd045799e.png)
I ask myselt if it's my fault so I tried with Unity Game Engine it's works. But why not Godot?
